OneCommunityGlobal/HGNRest
# Abi wbs tests

# Description
Full unit and integration test suite for WBS controller and routes.

## Related PRS (if any):
N/A

## Main changes explained:
- Unit testing controller methods
- Integration testing routes
- 
## How to test:
1. check into current branch
2. do `npm install` 
3. Run `npm run test wbs`
4. verify all tests pass

## Screenshots or videos of changes:
![Screenshot 2024-05-07 at 3 10 06 PM](https://github.com/OneCommunityGlobal/HGNRest/assets/59585815/af7fedb6-f1f0-4fb7-832b-27fe068c3927)

## Note:
Include the information the reviewers need to know.
